abi.html

```diff
@@ -4729,6 +4730,13 @@
 </p>
 
 <p>
+When an exception-specification (i.e., "noexcept", "noexcept(expression)", or
+"throw(type(s))") is part of the function type, it is mangled according to
+<code>&lt;<a href="#mangle.exception-spec">exception-spec</a>&gt;</code> as
+described below.
```

Comment:
Worth mentioning that a non-instantiation-dependent, potentially-throwing exception specification is not mangled?